changelog.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## NEXT
+
+- Added the method `from_dict` for all node and edges.
```

Comment:
Can you be a bit more specific, i.e. say what those classes are + hyperlink them with the full reference